### PR TITLE
Add new meets_version helper

### PR DESCRIPTION
**Pull request by banesullivan**
For https://github.com/pyvista/pyvista/pull/580

credit to @akaszynski

### CODE REVIEW

File: scooby/knowledge.py [hunk 1]
```diff
@@ -128,3 +128,53 @@ def get_standard_lib_modules():
...
+    >>> meets_version('0.25.1', '0.25.2')
+    False
+
+    >>> meets_version('0.26.0', '0.25.2')
+    True
+    """
+    va = version_tuple(version)
+    vb = version_tuple(meets)
+
+    if len(va) != len(vb):
```
Comment (akaszynski) [inline scooby/knowledge.py]: May want to add a trailing "0" if the version length is less than 3.  That way "0.2" can be compared to "0.2.1"

Comment (banesullivan) [reply scooby/knowledge.py]: ah good point

**Comment by akaszynski** [comment]
@banesullivan,

Please merge and version bump scooby for https://github.com/pyvista/pyvista/pull/580

**Comment by banesullivan** [comment]
Thanks for finishing this up, @akaszynski. I'll version bump scooby

**Pull request merged**